### PR TITLE
Updated 123-Reg.co.uk to include TOTP Support

### PR DESCRIPTION
**Pull request by dalDevelo**
Updated 123-Reg.co.uk to include TOTP Support + Support URL for 2FA setup

Screenshot 2024-08-31 at 01 10 20

Apologies for the dupes 🤦🏻‍♂️ - Was unable to re-open.
[#8180 ](https://github.com/2factorauth/twofactorauth/pull/8180)
[#8181 ](https://github.com/2factorauth/twofactorauth/pull/8181)

### CODE REVIEW

**Comment by hkamran80** [comment]
> Apologies for the dupes 🤦🏻‍♂️ - Was unable to re-open.

No worries. I think you may have been unable to re-open them because the branch was deleted.

File: entries/1/123-reg.co.uk.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
   "123 Reg": {
     "domain": "123-reg.co.uk",
-    "contact": {
-      "facebook": "123regfans",
-      "twitter": "123reg"
-    },
+    "tfa": [
+      "totp"
```
Comment (hkamran80) [inline entries/1/123-reg.co.uk.json]:
```suggestion
      "sms",
      "totp",
      "u2f"
```

According to the documentation page, they support SMS and FIDO2 security keys as well.

**Review by hkamran80 — CHANGES_REQUESTED** [review]

**Comment by dalDevelo** [comment]
Thank you, I'll try to be more thorough next time. I appreciate the edit 🙏🏼

**Review by hkamran80 — APPROVED** [review]

**Comment by hkamran80** [comment]
Perfect, thank you! Happy to help!

**Pull request merged**